src/components/points/CroupierPoints/CroupierPoints.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { PointsWrapper } from "../PointsElements";
+
+import pointsHelpers from "../PointsHelper";
+
+import useGameContext from "../../../hooks/useGameContext";
+import { rules } from "../../../const/rules";
+import { GameState } from "../../../const/gameState";
+import { useCroupierPoints } from "./CroupierPointsHooks";
+
+function CroupierPoints() {
+  const { croupierCards, playerCards, currentRoundStatus, setCurrentRoundStatus, drawOneCard } = useGameContext();
```

Comment:
cały czas się zastanawiam, czy nie lepszym pomysłem jest przeniesienie tego do hooka, głównie pod względem testów, ale jako, że ten hook jest tylko upiekszęniem to wrzuciłbym całość do hooka podobnie jak zrobiłeś w useRoundHistory